static/styles.css

```diff
@@ -74,6 +74,8 @@ address {
 
 details > summary {
   list-style: none;
+  display: flex;
```

Comment:
@bridgetburke adjusting the summary so the disclosure triangles align on the right

Reply:
This looks really nice for the filters. I just realized that this might not work as well for the collapsed Copyright section because the triangle is too far over and small. May need an outline or border for clickability if we use it there. 

<img width="1239" alt="Screen Shot 2021-09-21 at 10 16 22 AM" src="https://user-images.githubusercontent.com/29953622/134187682-5e0c23df-0440-41b2-9ac3-e86f27cefef5.png">